apps/i18n/common/en_us.json

```diff
@@ -1988,7 +1988,9 @@
   "verifyYear": "I plan on teaching Unit 6 (Physical Computing) of CS Discoveries to a classroom of at least 10 students by the end of the '20-'21 school year.",
   "version": "Version",
   "versionHistory_ajaxFailure": "An error occurred.",
-  "versionHistory_clearProgress_prompt": "Are you sure you want to clear all progress for this level?",
+  "versionHistory_clearProgress_header": "Clear your code and start over?",
+  "versionHistory_clearProgress_prompt": "Are you sure you want to restart this level? This will clear all of your code.",
+  "versionHistory_clearProgress_templateLevelWarning": "Please note - this level shares code with other levels, so starting over here will also clear your code on the other levels.",
```

Comment:
I have a feeling this may not be enough context for a student, I imagine a student asking, which other levels? But I can't think of a better option

Reply:
Yeah, I agree but it's currently difficult to tell which other levels are linked.  This is definitely not ideal but better than nothing?